plugin/web/extensions/location/modals/terrainModal/index.tsx

```diff
@@ -0,0 +1,22 @@
+import useHook from "@web/extensions/location/hook";
+import { styled } from "@web/theme";
+
+import CommonModalWrapper from "../commonModalWrapper";
+
+const TerrainModal: React.FC = () => {
+  const { handleTerrainModalChange } = useHook();
+
+  return (
+    <CommonModalWrapper title="地形データ" onModalChange={handleTerrainModalChange}>
+      <>
```

Comment:
Test it, but I don't think you need the Fragments

Reply:
386626f0ec753cdcae932611366d0e17a73470a4

Reply:
the fragments removed as discussed in previous comments